docs/Examples/BbhInitialData/BbhInitialData.ipynb

```diff
@@ -0,0 +1,463 @@
+      "/VolumeData.vol/ObservationId1423324405707320626/ExtrinsicCurvature_xx Dataset {61344}\n",
+      "/VolumeData.vol/ObservationId1423324405707320626/ExtrinsicCurvature_yx Dataset {61344}\n",
+      "/VolumeData.vol/ObservationId1423324405707320626/ExtrinsicCurvature_yy Dataset {61344}\n",
+      "/VolumeData.vol/ObservationId1423324405707320626/ExtrinsicCurvature_zx Dataset {61344}\n",
+      "/VolumeData.vol/ObservationId1423324405707320626/ExtrinsicCurvature_zy Dataset {61344}\n",
+      "/VolumeData.vol/ObservationId1423324405707320626/ExtrinsicCurvature_zz Dataset {61344}\n",
+      "/VolumeData.vol/ObservationId1423324405707320626/HamiltonianConstraint Dataset {61344}\n",
+      "/VolumeData.vol/ObservationId1423324405707320626/InertialCoordinates_x Dataset {61344}\n",
+      "/VolumeData.vol/ObservationId1423324405707320626/InertialCoordinates_y Dataset {61344}\n",
+      "/VolumeData.vol/ObservationId1423324405707320626/InertialCoordinates_z Dataset {61344}\n",
+      "/VolumeData.vol/ObservationId1423324405707320626/Lapse Dataset {61344}\n",
+      "/VolumeData.vol/ObservationId1423324405707320626/MomentumConstraint_x Dataset {61344}\n",
+      "/VolumeData.vol/ObservationId1423324405707320626/MomentumConstraint_y Dataset {61344}\n",
+      "/VolumeData.vol/ObservationId1423324405707320626/MomentumConstraint_z Dataset {61344}\n",
+      "/VolumeData.vol/ObservationId1423324405707320626/Shift_x Dataset {61344}\n",
+      "/VolumeData.vol/ObservationId1423324405707320626/Shift_y Dataset {61344}\n",
+      "/VolumeData.vol/ObservationId1423324405707320626/Shift_z Dataset {61344}\n",
+      "/VolumeData.vol/ObservationId1423324405707320626/SpatialMetric_xx Dataset {61344}\n",
+      "/VolumeData.vol/ObservationId1423324405707320626/SpatialMetric_yx Dataset {61344}\n",
+      "/VolumeData.vol/ObservationId1423324405707320626/SpatialMetric_yy Dataset {61344}\n",
+      "/VolumeData.vol/ObservationId1423324405707320626/SpatialMetric_zx Dataset {61344}\n",
+      "/VolumeData.vol/ObservationId1423324405707320626/SpatialMetric_zy Dataset {61344}\n",
+      "/VolumeData.vol/ObservationId1423324405707320626/SpatialMetric_zz Dataset {61344}\n",
+      "/VolumeData.vol/ObservationId1423324405707320626/bases Dataset {696}\n",
+      "/VolumeData.vol/ObservationId1423324405707320626/connectivity Dataset {294400}\n",
+      "/VolumeData.vol/ObservationId1423324405707320626/grid_names Dataset {5272}\n",
+      "/VolumeData.vol/ObservationId1423324405707320626/quadratures Dataset {696}\n",
+      "/VolumeData.vol/ObservationId1423324405707320626/total_extents Dataset {696}\n",
+      "/src.tar.gz              Dataset {5301349}\n"
+     ]
+    }
+   ],
+   "source": [
+    "!h5ls -r BbhVolume*.h5"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We plot the diagnostics in `BbhReductions.h5` to see what happened during the\n",
```

Comment:
Do I need to understand all the code below on the first pass or should I skip to the figure right away? I think a few sentences guiding the reading through this relatively long chunk of low-level code would be good. E.g. `Below are some routines that read the residuals, split them into different iterations (both linear and nonlinear), then we plot the linear and nonlinear residual as a function of the cumulative linear iteration. Feel free to skip to the plot below and jump back to the reduction data parsing and plotting code later!`?